phpstan.neon

```diff
@@ -37,7 +37,7 @@ parameters:
     scanDirectories:
         - stubs
 
-    reportUnmatchedIgnoredErrors: false
+    reportUnmatchedIgnoredErrors: true
```

Comment:
should be false to make `composer phpstan-config` CI green, it seems because command `php bin/clean-phpstan.php` cut in the middle process so it not rolled back to false.